test/boards/nucleo_l4r5zi/flash_slot1

```diff
@@ -0,0 +1,6 @@
+#!/bin/env bash
+
+openocd=$(find ${ZEPHYR_BASE}/../../ -name openocd -type f | head -n 1)
+openocd_cfg=$(find ${ZEPHYR_BASE}/../../ -name st_nucleo_l4.cfg -type f | head -n 1)
+
+${openocd} -f ${openocd_cfg} -c "program ${1} 0x8040000 verify exit reset"
```

Comment:
Hardcoded address. Maybe use `hex` file which itself knows it's offset?

Reply:
`hex` is linked for **slot0**. here we need to put it to **slot1**